application/src/main/java/org/thingsboard/server/service/queue/DefaultTbClusterService.java

```diff
@@ -340,29 +342,33 @@ public void onDeviceAssignedToTenant(TenantId oldTenantId, Device device) {
 
     @Override
     public void onResourceChange(TbResourceInfo resource, TbQueueCallback callback) {
-        TenantId tenantId = resource.getTenantId();
-        log.trace("[{}][{}][{}] Processing change resource", tenantId, resource.getResourceType(), resource.getResourceKey());
-        TransportProtos.ResourceUpdateMsg resourceUpdateMsg = TransportProtos.ResourceUpdateMsg.newBuilder()
-                .setTenantIdMSB(tenantId.getId().getMostSignificantBits())
-                .setTenantIdLSB(tenantId.getId().getLeastSignificantBits())
-                .setResourceType(resource.getResourceType().name())
-                .setResourceKey(resource.getResourceKey())
-                .build();
-        ToTransportMsg transportMsg = ToTransportMsg.newBuilder().setResourceUpdateMsg(resourceUpdateMsg).build();
-        broadcast(transportMsg, callback);
+        if (resource.getResourceType() == ResourceType.LWM2M_MODEL) {
+            TenantId tenantId = resource.getTenantId();
+            log.trace("[{}][{}][{}] Processing change resource", tenantId, resource.getResourceType(), resource.getResourceKey());
+            TransportProtos.ResourceUpdateMsg resourceUpdateMsg = TransportProtos.ResourceUpdateMsg.newBuilder()
+                    .setTenantIdMSB(tenantId.getId().getMostSignificantBits())
+                    .setTenantIdLSB(tenantId.getId().getLeastSignificantBits())
+                    .setResourceType(resource.getResourceType().name())
+                    .setResourceKey(resource.getResourceKey())
+                    .build();
+            ToTransportMsg transportMsg = ToTransportMsg.newBuilder().setResourceUpdateMsg(resourceUpdateMsg).build();
+            broadcast(transportMsg, DataConstants.LWM2M_TRANSPORT_NAME, callback);
+        }
     }
 
     @Override
     public void onResourceDeleted(TbResourceInfo resource, TbQueueCallback callback) {
-        log.trace("[{}] Processing delete resource", resource);
-        TransportProtos.ResourceDeleteMsg resourceUpdateMsg = TransportProtos.ResourceDeleteMsg.newBuilder()
-                .setTenantIdMSB(resource.getTenantId().getId().getMostSignificantBits())
-                .setTenantIdLSB(resource.getTenantId().getId().getLeastSignificantBits())
-                .setResourceType(resource.getResourceType().name())
-                .setResourceKey(resource.getResourceKey())
-                .build();
-        ToTransportMsg transportMsg = ToTransportMsg.newBuilder().setResourceDeleteMsg(resourceUpdateMsg).build();
-        broadcast(transportMsg, callback);
+        if (resource.getResourceType() == ResourceType.LWM2M_MODEL) {
+            log.trace("[{}] Processing delete resource", resource);
```

Comment:
in the log , on the first place should be tenantid, than everything else
An example above is much better
```
log.trace("[{}][{}][{}] Processing change resource", tenantId, resource.getResourceType(), resource.getResourceKey());
```